coordinator/tests/services/translation/common/getResourcesOfType.spec.ts

```diff
@@ -0,0 +1,28 @@
+import * as TestResources from "../../../resources/test-resources"
+import {
+  getMedicationRequests,
+  getPatient,
+  getProvenances
+} from "../../../../src/services/translation/common/getResourcesOfType"
+
+describe("getResourcesOfType", () => {
+  const bundle = TestResources.examplePrescription1.fhirMessageUnsigned
+
+  test("getMedicationRequests", () => {
+    const medicationRequest = getMedicationRequests(bundle)
+
+    medicationRequest.map((medicationRequest) => expect(medicationRequest.resourceType).toBe("MedicationRequest"))
+  })
+
+  test("getPatient", () => {
+    const patient = getPatient(bundle)
+
+    expect(patient.resourceType).toBe("Patient")
+  })
+
+  test("getProvenance", () => {
+    const provenances = getProvenances(bundle)
+
+    provenances.map((provenance) => expect(provenance.resourceType).toBe("Pro"))
```

Comment:
ResourceType should be "Provenance", and the bundle here should be fhirMessageSigned from TestResources

Reply:
Before the refactor we were asserting the length of the array returned, maybe we should add this check back in to ensure we're not getting any false positives for tests as in this case

Reply:
ahh yeah this is something i left and never came back to, thanks for spotting!

Reply:
maybe we should have the function throw an error if it doesnt bring back a resource? can chat about it

Reply:
discussed and agreed that moving the expectations of number of entries returned for a resource type made sense to be asserted during validation